changelog.md

```diff
@@ -20,6 +20,7 @@
 * LLT-4128: Add integration tests for IPv6 MagicDNS service
 * LLT-3316: Improve usage of peer-reflexive endpoints
 * LLT-4246: Add self nordname as well as self IP address to the DNS records
+* LLT-4179: Fix problem with STUN session failure in tests
```

Comment:
Well, in tests it was spotted, I guess it was also more visible, because we have this fake Derp server so the connection took even longer, but I guess it happened as well anywhere, just people usually read logs from the tests

Reply:
yeah okay, but did the original change made meshnet connection any different(would the direct connection be made faster with this maybe?). I just don't think changelog of "fixed stun test" is very useful

Reply:
@baltrus1 changed the changelog entry, maybe the new one sounds better to you?

Reply:
again, it's just a nitpick, I'm not even sure if we have such rules for changelog entries

Reply:
Well, I consulted that with @Jauler and according to him this task should have some changelog entry

Reply:
yep the new one sounds much better! thank you